docs/spark-queries.md

```diff
@@ -318,12 +318,20 @@ To show a table's current partitions:
 SELECT * FROM prod.db.table.partitions
 ```
 
-| partition | record_count | file_count |
-| -- | -- | -- |
-|  {20211001, 11}|           1|         1|
-|  {20211002, 11}|           1|         1|
-|  {20211001, 10}|           1|         1|
-|  {20211002, 10}|           1|         1|
+If this table is not partitioned    
```

Comment:
Im not sure about adding non-partitioned example for "partitions" table , as I think its few users would think to do this, but it is right under the header so most people have to scroll past to see what they really want to see (the schema).  Id suggest removing it for now, which would be consistent with the other tables, if it sounds reasonable to you?

Reply:
Yeah, I I keep the result of partitioned table and display the result of non-partitioned table using note to make it more understandable.

Reply:
addressed it.